### PR TITLE
NP-9359 Changes to the address fields displayed  in the Void Check flow

### PR DESCRIPTION
**Pull request by xuerongNanopay**
ticket: https://nanopay.atlassian.net/browse/NP-9359

### CODE REVIEW

File: src/foam/nanos/auth/Address.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ foam.CLASS({
     { name: 'INVALID_ADDRESS_1', message: 'Invalid value for address line 1' },
     { name: 'INVALID_POSTAL_CODE', message: 'Valid Postal Code or ZIP Code required' },
     { name: 'POSTAL_CODE_REQUIRE', message: 'Postal Code required' },
-    { name: 'STREET_NAME_REQUIRED', message: 'Street name required' },
+    { name: 'STREET_NAME_REQUIRED', message: 'Street Name required' },
```
Comment (sarthak-marwaha) [inline src/foam/nanos/auth/Address.js]: This message can be removed if you set required:true on the streetName field and remove the validationPredicates from it

Comment (xuerongNanopay) [reply src/foam/nanos/auth/Address.js]: updated

**Review by Nauna — APPROVED** [review]
I don't think validation should have to do with Structured vs unstructured - but after seeing this, wonder what else lurks in our code.

**Comment by jinnnn9** [comment]
> I don't think validation should have to do with Structured vs unstructured - but after seeing this, wonder what else lurks in our code.

Actually it does. Unstructured address is modelled in a way so that it does not require street name and street number. If you look at UnstructuredAddressView.js, you can see that the view does not render both of them. I would suggest the street name validation to be similar to that of street number.

**Pull request merged**